src/act/node/canister_method/query_method.rs

```diff
@@ -57,9 +57,9 @@ impl Deref for QueryMethod {
     }
 }
 
-impl Declare<Vec<String>> for QueryMethod {
-    fn to_declaration(&self, keyword_list: &Vec<String>, _: String) -> Option<Declaration> {
-        let function_declaration = self.generate_function_body(keyword_list);
+impl Declare<Context> for QueryMethod {
```

Comment:
I'm wondering about making Declare be always for Context instead of using generics. Because right now that's all we have, but I guess there is a world where different things have different contexts even though we don't currently live in that world.